Project.toml

```diff
@@ -28,6 +28,7 @@ FiniteDifferences = "0.9, 0.10, 0.11"
 HybridArrays = "0.3"
 LightGraphs = "1"
 ManifoldsBase = "0.9.6"
+RecipesBase = "1"
```

Comment:
Hmm... you perhaps should also add `Colors` to `deps`? And in the tests you should at least make sure that `include("recipes.jl")` is executed and parsed without errors so you should in some test file do `using RecipesBase`. Maybe you could also do something similar to https://github.com/JuliaPlots/RecipesBase.jl/blob/21ebce5df7a19c5097ddac79a224b1503d02cc8e/test/runtests.jl#L23 for testing.

Reply:
Sure, I just did not yet have time to check for a proper testing, but loading it and testing that it is only loaded when needed belongs to the tests.

Sure, I can also add Colors to deps.